src/techonomy/django/django_tableselect/static/django_tableselect/tableselect.js

```diff
@@ -0,0 +1,123 @@
+// TODO: make this code compatible with dinosaurs (old browsers)
+addEventListener("DOMContentLoaded", () => {
+  const tableSelector = "[data-tableselect]";
+  const bulkCheckboxSelector = "input[type=checkbox][data-bulk-select]";
+  const ariaNarratorSelector = "[data-tableselect-narrator]";
+  const selectCheckboxSelector = "input[type=checkbox][name]";
+
+  const allSelectedTextAttribute = "data-trans-all-selected";
+  const selectAllSelectedTextAttribute = "data-trans-select-all";
+  const deselectAllSelectedTextAttribute = "data-trans-deselect-all";
+  const numSelectedTextAttribute = "data-trans-num-selected";
+
+  const allChecked = (nodes) => Array.from(nodes).every((node) => node.checked);
+  const partiallyChecked = (nodes) =>
+    Array.from(nodes).some((node) => node.checked);
+
+  const tables = document.querySelectorAll(tableSelector);
+
+  tables.forEach((table) => {
+    const bulkCheckbox = table.querySelector(bulkCheckboxSelector);
+    const selectCheckBoxes = table.querySelectorAll(selectCheckboxSelector);
```

Comment:
const checkboxes i.p.v. selectCheckboxes. Ze zijn hier nog niet gecheckt of selected.

Reply:
`select` slaat niet op het feit dat ze gecheckt of niet gecheckt zijn. Het slaat op de checkboxes van de rijen.

Ik maak er wel het volgende van:
- bulkCheckbox -> selectAllCheckbox
- selectCheckBoxes -> rowCheckboxes